Jenkinsfile

```diff
@@ -918,6 +912,44 @@ pipeline {
                 Navi21_build_cmd = "LLVM_PATH=/opt/rocm/llvm CTEST_PARALLEL_LEVEL=2 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 MIOPEN_LOG_LEVEL=5 make -j\$(nproc) check"
             }
             parallel{
+                stage('dbsync gfx908') {
+                    when {
+                        beforeAgent true
+                        expression { params.TARGET_GFX908 }
+                    }
+                    options {
+                        retry(2)
+                    }
+                    agent{ label rocmnode("gfx908") }
+                    environment{
+                        setup_flags="-DMIOPEN_TEST_DBSYNC=1"
+			config_targets='test_db_sync'
```

Comment:
@JehandadKhan What is the purpose of thinks like
```
config_targets='test_db_sync'
```
and then
```
buildHipClangJobAndReboot(... config_targets: config_targets ...)
```

What if we'll simply
```
buildHipClangJobAndReboot(... config_targets: 'test_db_sync' ...)
```
?